tests/gui/conversation/export/test_device.py

```diff
@@ -95,60 +67,26 @@ def test_Device_run_print_file(mocker, session, homedir):
         assert len(print_requested_emissions) == 1
 
 
-def test_Device_run_print_file_not_qubes(mocker, session, homedir):
```

Comment:
All the `test_*_not_qubes` are now unnecessary because the corresponding code is gone.